packages/synapse-react-client/stories/LastLoginInfo.stories.ts

```diff
@@ -0,0 +1,46 @@
+import { Meta, StoryObj } from '@storybook/react'
+
+import LastLoginInfo from '../src/components/Authentication/LastLoginInfo'
+import { LoginMethods } from '../src/components/Authentication/LoginMethod'
+import { LOGIN_USERNAME_PASSWORD } from '../src/utils/SynapseConstants'
+
+const meta = {
+  title: 'Authentication/LastLoginInfo',
+  component: LastLoginInfo,
+  argTypes: {
+    lastLoginMethod: {
+      options: LoginMethods,
+      control: 'radio',
+    },
+    lastLoginDate: {
+      control: 'text',
+    },
```

Comment:
Prevent storybook from rendering this control as a date

Reply:
With suggested changes, it would probably be ok if this is a datepicker